membrane_rtc_engine_webrtc/lib/webrtc_endpoint.ex

```diff
@@ -513,11 +513,6 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
     {[notify_parent: {:forward_to_parent, {:media_event, media_event}}], state}
   end
 
-  @impl true
```

Comment:
We introduced this in https://github.com/jellyfish-dev/membrane_rtc_engine/pull/299

Could you please try to add and remove screensharing and see whether it works? Also, would be nice to dicuss what is going on here with this child removal as I lost myself a little bit

Reply:
That's true, but this callback was subsequently removed by you in the very next commit, https://github.com/jellyfish-dev/membrane_rtc_engine/commit/e9add7a1ed24f6bbbf58a2abf65d241c68d2fe7b 

I first added the callbacks both in the engine and the webrtc endpoint as part of the monorepo restructuring; during the handling of this issue I've realised that this callback was removed from the endpoint, so I chose to include this change in the same PR.

Screensharing in Videoroom seems to work fine.